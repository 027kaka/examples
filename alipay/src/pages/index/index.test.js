import * as React from 'react';
import renderer from 'react-test-renderer';
import App from '../../app';
import IndexPage from './';

test('renders correctly', () => {
  const tree = renderer
    .create(
      <App>
        <IndexPage />
      </App>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
