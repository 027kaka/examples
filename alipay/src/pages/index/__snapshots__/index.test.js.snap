// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<view
  class="page-todos"
>
  <view
    class="user"
  >
    <button
      class="login-button"
      hoverClass="none"
      onTap={[Function]}
    >
      <image
        background-size="cover"
        class="avatar"
        src="image-mock-path"
      />
    </button>
    <view
      class="nickname"
    >
      My
       Todo List
      <text
        class="login-tip"
      >
        (Tap to login â†‘)
      </text>
    </view>
  </view>
  <view
    class="todo-items"
  >
    <checkbox-group
      class="todo-items-group"
      onChange={[Function]}
    >
      <label
        class="todo-item checked"
      >
        <checkbox
          checked={true}
          class="todo-item-checkbox"
          value={1}
        />
        <text
          class="todo-item-text"
        >
          Learning Javascript
        </text>
      </label>
      <label
        class="todo-item checked"
      >
        <checkbox
          checked={true}
          class="todo-item-checkbox"
          value={2}
        />
        <text
          class="todo-item-text"
        >
          Learning ES2016
        </text>
      </label>
      <label
        class="todo-item"
      >
        <checkbox
          checked={false}
          class="todo-item-checkbox"
          value={3}
        />
        <text
          class="todo-item-text"
        >
          Learning Remax
        </text>
      </label>
    </checkbox-group>
  </view>
  <view
    class="todo-footer"
  >
    <button
      class="add-button"
      hoverClass="none"
      onTap={[Function]}
    >
      <text
        class="add-icon"
      >
        +
      </text>
      <text>
        Add Todo
      </text>
    </button>
  </view>
</view>
`;
